src/ripplegw/waveforms/IMRPhenomXAS.py

```
@@ -1516,9 +1514,10 @@ def gen_IMRPhenomXAS_hphc(f: Array, params: Array, f_ref: float):
       hc (array): Strain of the cross polarization
     """
     iota = params[7]
-    h0 = gen_IMRPhenomXAS(f, params, f_ref)
+    h22 = gen_IMRPhenomXAS(f, params, f_ref)
 
-    hp = h0 * (1 / 2 * (1 + jnp.cos(iota) ** 2))
-    hc = -1j * h0 * jnp.cos(iota)
+    # h+ = -h22 · (1+cos²ι)/2  (LAL sign convention: h+ = −h_lm · antenna)
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `ι` (GREEK SMALL LETTER IOTA). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF003)